### PR TITLE
Fix some raiders missing swimspeed modifiers

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Fix some raiders missing swimspeed modifiers and fix all their values
Fix some of our entities not using our custom movement controller
Fix our entities beeing affected by bubble columns

Review please

### CODE REVIEW

File: src/api/java/com/minecolonies/api/entity/mobs/vikings/AbstractEntityNorsemen.java [hunk 1]
```diff
@@ -20,6 +20,11 @@
  */
 public abstract class AbstractEntityNorsemen extends AbstractEntityMinecoloniesMob
 {
+    /**
+     * Swim speed for pirates
+     */
+    private static final double PIRATE_SWIM_BONUS = 1.5;
```
Comment (ravenbuilder934) [inline src/api/java/com/minecolonies/api/entity/mobs/vikings/AbstractEntityNorsemen.java]: are they pirates though?

Comment (someaddons) [reply src/api/java/com/minecolonies/api/entity/mobs/vikings/AbstractEntityNorsemen.java]: sure^^

File: src/api/java/com/minecolonies/api/entity/mobs/AbstractEntityMinecoloniesMob.java [hunk 2]
```diff
@@ -235,6 +235,8 @@ public AbstractAdvancedPathNavigate getNavigation()
         return newNavigator;
     }
 
+    public abstract double getSwimSpeedFactor();
```
Comment (Raycoms) [inline src/api/java/com/minecolonies/api/entity/mobs/AbstractEntityMinecoloniesMob.java:243]: no javadoc =O

**Review by Raycoms — DISMISSED** [review]

File: src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java [hunk 3]
```diff
@@ -734,4 +740,24 @@ public void updatePose(final Pose pose)
...
+    {
+        return false;
+    }
+
+    /**
+     * Do not allow bubble movement
+     *
+     * @param down
+     */
+    public void onInsideBubbleColumn(boolean down)
```
Comment (Raycoms) [inline src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java:810]: Maybe we want to allow upward movement and not downward?

Comment (someaddons) [reply src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java:810]: they do swim upwards by themselves, they just no longer will be hindered in where they want to move by the column itself

**Comment by Raycoms** [comment]
Just missing the conflicts then :shipit:

**Pull request closed without merging**